### PR TITLE
feat(jsonld): Add method to find node in document

### PR DESCRIPTION
**Pull request by ender503**
The DID public key information may be embedded in the DID document,
so add the "findNodeById()" function to find the node in the
document.

See also: https://w3c-ccg.github.io/did-spec/#public-keys

### CODE REVIEW

File: packages/jsonld/README.md [hunk 1]
```diff
@@ -148,6 +150,18 @@ Performs JSON-LD flattening.
 Performs JSON-LD framing.
 
 **Returns**: <code>Promise.&lt;object&gt;</code> - Promise that resolves to the framed output.  
+<a name="module_jsonld..findNodeById"></a>
+
+### *jsonld*~findNodeById(document, id)
+
+| Param | Description |
```
Comment (jserv) [inline packages/jsonld/README.md]: Should be "Parameter".

File: packages/jsonld/src/jsonld.ts [hunk 2]
```diff
@@ -123,3 +123,29 @@ export const frame = async (input: object, frame: object, options: object = {}):
...
+ * @function findNodeById
+ * @param document - the JSON-LD document to find.
+ * @param id - String that specifies the ID value.
+ * @return {Promise<object|null>} Promise that resolves to the node.
+ */
+export const findNodeById = async (document: object, id: string): Promise<object | null> => {
+  const flattened: { [index: string]: any } = await flatten(document, {});
+
+  if (!flattened.hasOwnProperty('@graph')) {
+    throw new Error('@graph property is not in the flattened document');
```
Comment (jserv) [inline packages/jsonld/src/jsonld.ts]: Change "is not" to "does not exist".

Comment (ender503) [reply packages/jsonld/src/jsonld.ts]: Fixed.

File: packages/jsonld/README.md (hunk 1 above)
Comment (ender503) [reply packages/jsonld/README.md]: This table is generated by the `jsdoc2md` with the default style and `iota.js` is using the same [style](https://github.com/iotaledger/iota.js/tree/next/packages/core#corecomposeapisettings).
I think it's okay to keep this table style.
If it still looks weird, I'll send another pull request to customize the table style.

**Pull request merged**